app/controllers/api/v3/timelines/for_you_controller.rb

```diff
@@ -6,96 +6,20 @@ class Api::V3::Timelines::ForYouController < Api::BaseController
   before_action :set_for_you_default, only: [:show]
   after_action :insert_pagination_headers, only: [:show], unless: -> { @statuses.empty? }
 
-  rescue_from PersonalForYou::Error do |exception|
-    render json: { error: exception }, status: 404
-  end
-
-  def index
-    result = PersonalForYou.new.mammoth_user_profile(acct_param)
-    render json: result
-  end
-
   def show
-    @statuses = set_for_you_feed
+    @statuses = list_statuses
     render json: @statuses,
            each_serializer: REST::StatusSerializer
   end
 
-  # When Updating User with new settings
-  # Update status to 'pending'
-  # Also need to trigger a clear & rebuild of their personal for you feed
-  def update
-    payload = for_you_params
-    payload[:status] = 'pending'
-    result = PersonalForYou.new.update_user(acct_param, payload)
-
-    # Set Queue specificly for a rebuild
-    UpdateForYouWorker.set(queue: 'mammoth_critial').perform_async({ acct: acct_param, rebuild: true })
-    render json: result
-  end
-
   private
 
   def set_for_you_default
     @default_owner_account = Account.local.where(username: FOR_YOU_OWNER_ACCOUNT).first!
     @account = account_from_acct
-    @user = user_from_param
-    @is_beta_program = beta_param
-  end
-
-  # Check and see if they're a Mammoth User
-  # If they are get their foryou feed
-  # Otherwise send them MammothPicks
-  def set_for_you_feed
-    should_personalize = validate_mammoth_account
-    if should_personalize
-      # Getting personalized
-      fufill_foryou_statuses
-    else
-      # Getting the public feed
-      enroll_beta
-      cached_list_statuses
-    end
-  end
-
-  # Check account_from_acct finds an account
-  # Check AccountRelay that they are a Mammoth 2.0 User
-  # @return [Boolean]
-  def validate_mammoth_account
-    return false if @account.nil?
-
-    PersonalForYou.new.mammoth_user?(acct_param)
-  end
-
-  # Only checking for beta parameter
-  # After we've validated the acct is NOT on the beta list
-  # So if you're already on the beta list we're not going add them
-  def enroll_beta
```

Comment:
Just confirming that we no longer need this enroll_beta call

Reply:
yep. beta enrollment is a temporary list that we were using to run personalization on TestFlight users when trending follows and friends of friends were first priority. with the waitlist enabled and working, this is no longer needed